MiniApp/Classes/core/Models/MiniAppManifest.swift

```diff
@@ -31,6 +32,32 @@ internal struct MACustomPermissionsResponse: Decodable {
     }
 }
 
+public struct MASDKAccessTokenPermission: Codable, Equatable, Hashable {
+    public var audience: String
+    public var scopes: [String]
+
+    private enum CodingKeys: String, CodingKey {
+        case audience,
+             scopes
+    }
+
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(audience)
+    }
+
+    public static func == (lhs: MASDKAccessTokenPermission, rhs: MASDKAccessTokenPermission) -> Bool {
+        lhs.audience == rhs.audience && rhs.scopes.sorted() == lhs.scopes.sorted()
+    }
+
+    public func isPartOf(_ fullScopes: [MASDKAccessTokenPermission]) -> Bool {
```

Comment:
Some documentation for public interfaces would be great to have

Reply:
of course. I just thought the name was self explanatory ^^

Reply:
Could you also add documentation for the class `MASDKAccessTokenScopes` too?

Reply:
Already planned it @corycaywood :)